src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square($size) {
```

Comment:
The mixin name 'square' could be more descriptive related to the calendar, such as 'calendarCellSize' or 'dayCellSize'.